src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10",
+    "System.Net.Sockets": "4.0.10-beta-*",
```

Comment:
@stephentoub The compilation with 4.0.10.0 goes well but we get runtime errors with 4.0.10.0 while the System.Net.Sockets is being looked for. 
For now I will leave this as is and I will open an issue to migrate this version to a higher version after testing.